docs/gettingstarted/index.md

```diff
@@ -1,60 +1,119 @@
-* To update the run configurations - replace all occurrences of `examplemod` to the mod id of your mod.
+```gradle
+// In some build.gradle
+archivesBaseName = 'mymod'
+```
 
-### Migration to Mojang's Official Mappings
+#### Group Id
 
-Forge uses Mojang's Official Mappings, or MojMaps, for the forseeable future. The official mappings provide class, method, and field names. Parameters and javadocs are not provided by this mapping set. Currently, there is no guarantee that these mappings are legally safe; however, Forge has decided to adopt them in good faith since Mojang wants them to be used. You can read about [Forge's stance here][mojmap].
+The `group` property should be set to your [top-level package][packaging], which should either be a domain you own or your email address:
+
+Type      | Value             | Top-Level Package
+:---:     | :---:             | :---
+Domain    | example.com       | `com.example`
+Subdomain | example.github.io | `io.github.example`
+Email     | example@gmail.com | `com.gmail.example`
+
+```gradle
+// In some build.gradle
+group = 'com.example'
+```
+
+The packages within your java source (`src/main/java`) should also now conform to this structure, with an inner package representing the mod id:
+
+```text
+com
+- example (group package)
+  - mymod (the mod id)
+    - MyMod.java (renamed ExampleMod.java)
+```
+
+#### Version
+
+Set the `version` property to the current version of your mod. We recommend using a [variation of semantic versioning][semver].
+
+```gradle
+// In some build.gradle
+version = '1.19.4-1.0.0.0'
+```
+
+### Additional Configurations
+
+Additional configurations can be found on the [ForgeGradle] docs.
 
 Building and Testing Your Mod
 -----------------------------
 
-1. To build your mod, run `gradlew build`. This will output a file in `build/libs` with the name `[archivesBaseName]-[version].jar`. This file can be placed in the `mods` folder of a Forge enabled Minecraft setup or distributed.
-2. To test run your mod, the easiest way is to use the run configs that were generated when you set up your project. Otherwise, you can run `gradlew runClient`. This will launch Minecraft from the `<runDir>` location along with your mod's code in any source sets specified within your run configurations. The default MDK includes the `main` source set, so any code written within `src/main/java` will be applied.
-3. You can also run a dedicated server using the server run config or via `gradlew runServer`. This will launch the Minecraft server with its GUI. After the first run, the server will shut down immediately until the Minecraft EULA is accepted by editing `run/eula.txt`. Once accepted, the server will load and can be accessed via a direct connect to `localhost`.
+1. To build your mod, run `gradlew build`. This will output a file in `build/libs` with the name `[archivesBaseName]-[version].jar`. This file can be placed in the `mods` folder of a Forge-enabled Minecraft setup or distributed.
```

Comment:
```suggestion
1. To build your mod, run `gradlew build`. This will output a file in `build/libs` with the name `[archivesBaseName]-[version].jar`, by default. This file can be placed in the `mods` folder of a Forge-enabled Minecraft setup or distributed.
```